### DISCUSSION TITLE
Design System/Guidelines for Pre-Login Twilio.com Experience

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#3023] [General] [answered]
I am working on a few pages related to the pre-login Support Centers for Twilio.com (support.twilio.com).  I was just wondering, is there any documentation/design system/Figma libraries for the pre-login Twilio.com pages?

**Comment by kelseycavitt** [comment, marked as answer]
Hi @tonge495!

We currently don’t have explicit documentation or a system for just the pre-login [Twilio.com](http://twilio.com/) pages.
It may be helpful to chat with someone from Brand on how to proceed, as the branding on the various pre-login pages look slightly different from Paste. What we can advise is to use Paste. Additionally, we're currently in the process of implementing design language updates in Paste with a new Twilio theme that is much more brand-friendly. To learn more about these updates, you can check out our [updates guide](https://docs.google.com/document/d/1RACJwSG78YJtKMDgFY-VnbUzYyDqTae8nc3e-Y9a9Ho/edit?usp=sharing) and a [presentation](https://www.figma.com/file/wAXdm0CAM3rnjn3RJsWb2v/One-Twilio-Updates-Shareout-for-Team-UX?node-id=0%3A1&t=EuJbpIpuuq7lXYLd-1) that includes images of how the new updates will look.

It also may be helpful to chat with Diya Agha, who’s on the Web Design team to get some alignment on how those pages should appear as well. 

Let us know if you have any more questions.

**Reply by tonge495** [reply]
Thank you for the response @kelseycavitt!  In the future, will we be following the brand-friendly design language updates using the Twilio theme or continue using the existing pre-login Twilio.com page?  If so, is there a timeline?  Asking as we are very much in the preliminary stages of a pre and post-login experience for our Support Centers and would like to stay aligned to whatever the future design standards will be.

**Reply by loreina** [reply]
It's strongly recommended to use Paste and the Twilio theme to unify the design language across brand and product. There's an outline of migration timelines per product available in our [updates guide](https://docs.google.com/document/d/1RACJwSG78YJtKMDgFY-VnbUzYyDqTae8nc3e-Y9a9Ho/edit?usp=sharing).

**Reply by tonge495** [reply]
Great thank you @loreina!
